perception/sub8_perception/nodes/buoy_finder.py

```diff
@@ -297,15 +302,18 @@ def get_best_contour(self, contours):
         TODO: Use smarter contour filtering methods, like checking this it is circle like
         '''
         if len(contours) > 0:
-            cnt = max(contours, key=cv2.contourArea)
-            area = cv2.contourArea(cnt)
-            if area < self.min_contour_area:
-                return None
-            M = cv2.moments(cnt)
-            cx = int(M['m10'] / M['m00'])
-            cy = int(M['m01'] / M['m00'])
-            tpl_center = (int(cx), int(cy))
-            return cnt, tpl_center, area
+            sort = sorted(contours, key=cv2.contourArea, reverse=True)
+            for cnt in sort:
```

Comment:
Can you add a little header comment for this loop? Also, if your measure of contour changes, it might not be so good to have the return statement inside of the loop.